src/Sql/Sql/ChangeLog.md

```diff
@@ -31,6 +31,7 @@ Fix New-AzSqlDatabaseSecondary cmdlet to check for PartnerDatabaseName existence
 ## Version 2.1.1
 * Update references in .psd1 to use relative path
 * Upgraded storage creation in Vulnerability Assessment auto enablement to StorageV2
+* Adding support for changing existing Sql Managed Instance hardware generation
```

Comment:
Changes for the upcoming release should go under the section titled "Upcoming Release".